js/redux/modules/companyReducer.js

```diff
@@ -76,6 +81,22 @@ export const getCompany = companyID => dispatch => {
     });
 };
 
+export const getAllCompanyProjects = companyID => dispatch => {
+  database
+    .collection("companys/" + companyID + "/projects")
+    .get()
+    .then(collection => {
+      const projects = [];
+      collection.forEach(doc => {
+        projects.push(doc.data());
+      });
+      dispatch(getAllProjectsInfo(projects));
+    })
+    .catch(function(error) {
+      console.log("Error getting document:", error);
```

Comment:
Manage error state in Redux store instead of console logging. This would allow you to update the UI and give the user feedback if something goes wrong.